NEWS

```diff
@@ -49,6 +49,8 @@ New Functionality
 Changed Functionality
 ---------------------
 
+- The traditional TSV Zeek logs are now valid UTF8 by default.
```

Comment:
May help to explicitly mention it's possible to revert to old behavior via `LogAscii::enable_utf_8`.